### ISSUE TITLE
Spelling review of guide

### ISSUE THREAD

**Issue by creyeknife** [O3Labs/SmartEco#3]
Second block, You can find _your_ the installation link
Delete 'your'

3rd block
Softwar > Software

The sentence before first images
'If you see this out then you can' 
Think it's meant to be output?

Before diving into all of the implementation details, (add comma) let's take a closer look at the architecture (lowercase A) of SmartEco

Diagram
Add spaces before 1 and 2. So it's the same as 3

The provider layer which comes next, (add comma) implements

Finally, (add comma) at (Spelling mistake 'ay') the top level, (add comma) we have hAPPS (dAPI and dAPPS usually has lower case?)

HAPPS fundamentally differ from DAPPS in that there is no expectation or need for all, (add comma) or even the majority

Rather, (add comma) they should use it where it clearly

Wallet DAPP image
No capitals on transaction or signature or provider

As a decentralized application, (add comma) the first thing (remove s from things) that you

You will (remove the word access) authenticate a user using their credentials

Let's (add apostrophe) now look at how to (add 'to') connect to the wallet using the decentralized API. First let's (add apostrophe) look at the general flow map.

As you can see, (add comma) when the user first opens your DAPP via the browser, (add comma) you must check to see if the user is in a SmartEco Evnironment (spelling).

structured JSON format similar to (Something like > similar to) this.

For instance, (add comma) if O3 was the provider environment

 it's (add apostrophe) is highly reccommended (spelling)

application to know their account details,(add comma) you will receive a json (JSON should be capitals?) object 

At this point, (add comma) you are considered to be in

First, (add comma) it will allow you to query 

However, (add comma) this identity doesn't just 

If you're feeling more adventurous, then you could use (add 'use') this address as the primary key

In the next section, (add comma) we will cover how to access  

However, (add comma) the complete list of available read methods is: (add : )

You'll note that the balance query flow in general, (add comma) is much simpler than 

Write methods are methods (those > methods) that allow you to alter

blockchain on behalf of the user. (comma > Full stop) There are two types of 

then the (delete A) transaction id will be returned to your application as a receipt.

After receiving the transaction ID from the wallet, you (it > you) can handle this in various (many > various) ways, to verify 

In summary, (add comma) we have now described the basic process

However, (add comma) we will now expand it into generic contract invokes

Now let's (add apostrophe) break down this function. (full stop) We are invoking the contract with scriptHash

methods located in the (add 'the')contract and passing in the argument "hello" which is of type string. (of to a? this doesn't make sense I don't think?)

You can handle this in anyway you (tou > you) choose.

For example, (add comma) you may want to reload the information presented i

The currently supported events in the DAPI are as follows: (Add : )

However the complete list of available read methods is (I think this sentence isn't meant to be here)

DISCONNECTED -> User (capital for User) has revoked
